packages/fx-core/resource/package.nls.json

```diff
@@ -1150,6 +1150,10 @@
   "driver.file.updateJson.description": "Add or update appsettings to JSON file.",
   "driver.file.updateJson.summary.withTarget": "The value for appsettings has been generated successfully to %s.",
   "driver.env.error.folderNotExist": "Folder %s does not exist.",
+  "driver.deploy.FailedListPublishingCredentialsRemoteError": "Failed to list publishing credentials due to remote service error. \nPlease check your account and try again later.",
+  "driver.deploy.error.restartWebAppError": "Failed to restart web app.\nPlease try to restart the web app manually if the app doesn't work properly.",
```

Comment:
"Failed to restart web app" has called out the reason for asking user to restart web app manually. Suggest to remove texts: "if the app doesn't work properly."

Reply:
In my test cases, it works properly even without restart the web app. Although it is stated in the documentation that a restart is required after deployment, I read Kudu's source code and found that they perform the restart operation automatically. But I don't know if there are more complex cases that do require a restart.